src/index.js

```diff
@@ -13,6 +13,10 @@ window.$ = window.jQuery = require('jquery')
 require('bootstrap/dist/js/bootstrap')
 window.patternfly = {}
 window.patternfly = require('patternfly/dist/js/patternfly')
+window.selectpicker = {}
```

Comment:
I would need to retry, but as IIRC it's a hack to allow lazy init of those libraries.
I would not consider it as a pattern, so for the selectpicker please first try without it. If it works, let's do not use it.